diff --git a/chrome/browser/notifications/notification_platform_bridge_mac.mm b/chrome/browser/notifications/notification_platform_bridge_mac.mm
index 818a55960feddc8dd86717673f3164388afd9a0d..0bd414094c8c69ee478d56f6bb47e3a59d1f6b97 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac.mm
@@ -92,6 +92,7 @@ bool IsPersistentNotification(
 // an expensive call. Instead use NotificationPlatformBridgeMac::SupportsAlerts
 // which caches this value.
 bool SupportsAlertsImpl() {
+  BRAVE_SUPPORTSALERTSIMPL_
   int32_t major, minor, bugfix;
   base::SysInfo::OperatingSystemVersionNumbers(&major, &minor, &bugfix);
   // Allow alerts on all versions except 10.15.0, 10.15.1 & 10.15.2.
@@ -240,6 +241,7 @@ void NotificationPlatformBridgeMac::Display(
   } else {
     NSUserNotification* toast = [builder buildUserNotification];
     [notification_center_ deliverNotification:toast];
+    BRAVE_DISPLAY_
   }
 }
 
