diff --git a/chrome/browser/resources/settings/site_settings/category_default_setting.js b/chrome/browser/resources/settings/site_settings/category_default_setting.js
index f1aeec3e53c7c7d196111d4260b6ae6a85b02895..054758ec2ba4ff7a7f4921c7133f36b55e0fc8fa 100644
--- a/chrome/browser/resources/settings/site_settings/category_default_setting.js
+++ b/chrome/browser/resources/settings/site_settings/category_default_setting.js
@@ -171,6 +171,7 @@ Polymer({
             this.categoryEnabled ? ContentSetting.ALLOW : ContentSetting.BLOCK);
         break;
       case ContentSettingsTypes.AUTOMATIC_DOWNLOADS:
+      case ContentSettingsTypes.AUTOPLAY:
       case ContentSettingsTypes.CAMERA:
       case ContentSettingsTypes.CLIPBOARD:
       case ContentSettingsTypes.FONT_ACCESS:
