diff --git a/chrome/browser/resources/settings/site_settings/category_default_setting.js b/chrome/browser/resources/settings/site_settings/category_default_setting.js
index 0a9d21082eedb5802c5a9f54847cb559658cd903..8186ab5d4c6a634b8c63420f37ab097e77e8cd4b 100644
--- a/chrome/browser/resources/settings/site_settings/category_default_setting.js
+++ b/chrome/browser/resources/settings/site_settings/category_default_setting.js
@@ -171,6 +171,7 @@ Polymer({
             this.categoryEnabled ? ContentSetting.ALLOW : ContentSetting.BLOCK);
         break;
       case ContentSettingsTypes.AUTOMATIC_DOWNLOADS:
+      case ContentSettingsTypes.AUTOPLAY:
       case ContentSettingsTypes.CAMERA:
       case ContentSettingsTypes.CLIPBOARD:
       case ContentSettingsTypes.FONT_ACCESS:
